README.md

```diff
@@ -189,11 +189,47 @@ $ git diff --word-diff
 <dt>How do I force a line break?</dt>
 <dd>
 
-You can add a hard line break with the `<br/>` element.
-Although CommonMark and other lightweight markup languages
-allow trailing spaces to indicate breaks between consecutive lines,
-this syntax is incompatible with
-editors that automatically strip trailing whitespace.
+Some lightweight markup languages provide a dedicated syntax for line breaks,
+some support line breaks using the inline HTML element `<br>`\*,
+and some do both.
+
+<dl>
+
+<dt>AsciiDoc</dt>
+<dd>**Syntax**: A trailing space followed by a plus sign (`+`)</dd>
+<dd>**Inline HTML**: Not supported</dd>
+
+<dt>CommonMark</dt>
+<dd>**Syntax**: A trailing backslash (`\`) \*\*</dd>
+<dd>**Inline HTML**: Supported</dd>
+
+<dt>Markdown</dt>
+<dd>**Syntax**: None</dd>
+<dd>**Inline HTML**: Supported</dd>
+
+<dt>MediaWiki</dt>
+<dd>**Syntax**: None</dd>
+<dd>**Inline HTML**: Supported</dd>
+
+<dt>OrgMode</dt>
```

Comment:
<https://orgmode.org/> uses "Org mode," not "OrgMode." (The logo and the main heading on the home page uses "Org Mode" with a capital M, but all other instances seem to be "Org mode" or just "Org.")